cfgov/v1/atomic_elements/organisms.py

```diff
@@ -139,9 +139,9 @@ class Table(blocks.StructBlock):
     ]))
 
     class Meta:
-        icon = 'form'
+        icon = None
         template = '_includes/organisms/table.html'
-        label = 'Table'
+        label = ' '
```

Comment:
@vccabral see [here](https://github.com/cfpb/cfgov-refresh/pull/2295#issuecomment-240485427); it's a way to (temporarily) (mostly) hide the Table option in the Wagtail admin.